src/App.vue

```diff
@@ -1,26 +1,124 @@
 <template>
-  <HelloWorld :msg="'Hello world!'"/>
+  <NavBar
+    :pages="pages"
+  />
+
+  <router-view />
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
-
+import NavBar from "./components/Nav/NavBar.vue";
 
 export default {
-  name: 'App',
+  name: "App",
   components: {
-    HelloWorld,    
-}
-}
+    NavBar,
+  },
+  data() {
+    return {
+      pages: [
+        {
+          link: { text: "Home", url: "/" },
+          pageTitle: "Home Page",
+          content: "This is the home content",
```

Comment:
The pageTitle and content are not used in the button from what I saw. Can we remove them from here to not cause confusion?

Reply:
done